lua/lspconfig/util.lua

```diff
@@ -107,10 +107,6 @@ M.path = (function()
     end
   end
 
-  local function path_join(...)
-    return table.concat(M.tbl_flatten { ... }, '/')
```

Comment:
Didn't even support `\` slashes so this was just cosmetic...

Reply:
Tbf even neovim's `vim.fs.joinpath` doesn't use `\` or provide the option to do that. I guess the usecase is being able to concatenate paths without wrapping a table around it? Idk.

Reply:
yeah and avoiding double slashes "//" (technically valid, but maybe some plugins/scripts break on it). And communicates "intention".

For Nvim core, avoiding `\` slashes is probably good, because it should be discouraged. Most of the time, `\` is cargo-culting.